ext/net/lib.deno_net.d.ts

```diff
@@ -136,8 +136,8 @@ declare namespace Deno {
     /** Make the connection not block the event loop from finishing. */
     unref(): void;
 
-    readonly readable: ReadableStream<Uint8Array<ArrayBuffer>>;
-    readonly writable: WritableStream<Uint8Array<ArrayBufferLike>>;
+    readonly readable: ReadableStream<Uint8Array>;
+    readonly writable: WritableStream<Uint8Array>;
```

Comment:
These changes seem unrelated and undesirable (Probably an error while resolving conflicts?). Let's revert.

```suggestion
    readonly readable: ReadableStream<Uint8Array<ArrayBuffer>>;
    readonly writable: WritableStream<Uint8Array<ArrayBufferLike>>;
```